pyproject.toml

```diff
@@ -30,12 +30,16 @@ dependencies = [
   "numpy>=1.20",
   "pycapnp==2.0.0b1",
   "asyncio-atexit==1.0.1",
+  "deprecation>=2.1.0",
```

Comment:
Where are these used?

Reply:
search for @deprecated in node.py
(It has only use use case so far)